app/src/main/java/com/github/se/project/ui/navigation/NavigationActions.kt

```diff
@@ -30,7 +30,7 @@ object Screen {
   const val ADD_LESSON = "Add Lesson Screen"
   const val TUTOR_MATCH = "Tutor matching Screen"
   const val EDIT_REQUESTED_LESSON = "Edit Requested Lesson"
-  const val EDIT_SCHEDULED_LESSON = "Edit Scheduled Lesson"
+  const val CONFIRMED_LESSON = "Confirmed Lesson"
```

Comment:
consider renaming it "Confirmed Lesson Screen" for consistency with other screens naming

Reply:
Good catch, I forgot it.